lib/SPIRV/SPIRVWriter.cpp

```diff
@@ -2232,6 +2232,16 @@ SPIRVValue *LLVMToSPIRV::transIntrinsicInst(IntrinsicInst *II,
     return BM->addExtInst(STy, BM->getExtInstSetId(SPIRVEIS_OpenCL), ExtOp, Ops,
                           BB);
   }
+  case Intrinsic::minnum: {
```

Comment:
Please add `minnum` to `LLVMToSPIRV::isKnownIntrinsic`.

Reply:
Right. Should I add `maxnum` as well with a separate commit, or a separate patch would be desirable?

Reply:
I think we can do `maxnum` with a separate commit in scope of this PR.

Reply:
Done.